unit/Makefile

```diff
@@ -96,17 +97,29 @@ CPROVER_LIBS =../src/ansi-c/ansi-c$(LIBEXT) \
 
 OBJ += $(CPROVER_LIBS) testing-utils/testing-utils$(LIBEXT)
 
-TESTS = unit_tests$(EXEEXT) \
-        miniBDD$(EXEEXT) \
+CATCH_TEST = unit_tests$(EXEEXT)
+N_CATCH_TESTS = $(shell \
+                  cat $$(find . -name "*.cpp" \
+                    -and -not -name "miniBDD_new.cpp" \
+                    -and -not -name "catch_example.cpp") | \
```

Comment:
Isn't this run though? Included in the Makefile.

Reply:
Can we maybe just remove that example by now?

Reply:
I'm in favour of removing it

Reply:
Yeah - docs probably make more sense that a live example!